lib/src/network/connection_manager.dart

```diff
@@ -24,7 +24,7 @@ class _ConnectionManager {
     DbCommand isMasterCommand = DbCommand.createIsMasterCommand(db);
     MongoReplyMessage replyMessage = await connection.query(isMasterCommand);
     _log.fine(() => replyMessage.documents[0].toString());
-    var master = replyMessage.documents[0]["ismaster"];
+    var master = replyMessage.documents[0]["ismaster"] == true;
```

Comment:
you can declare it as bool and remove the `== true` part.

Reply:
ideally `final bool master = ...`

Reply:
Didn't even realize that. In my head, this was a decent trick to coerce the value into a bool